src/Table/HeaderCell.jsx

```diff
@@ -25,14 +26,19 @@ export default function HeaderCell({
         sortable && cssClass.SORTABLE,
         className
       )}
-      noWrap
       onClick={() => sortable && onSortChange()}
       width={width}
     >
-      <div className={cssClass.LABEL}>
-        {children}
-      </div>
-      {sortable && <SortIcons direction={activeSortDirection} className={cssClass.SORT} />}
+      <FlexBox alignItems={ItemAlign.CENTER} >
```

Comment:
The flexbox is to keep the sort arrows on the same line as the label. Is there an easy way to do that without flex?